ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -11,11 +16,92 @@ export default function FeeCard({
   tokenApprovalTextComponent,
   tokenApprovalSourceTokenSymbol,
   onTokenApprovalClick,
+  metaMaskFee,
+  savings,
+  isBestQuote,
+  numberOfQuotes,
+  onQuotesClick,
+  conversionRate,
+  currentCurrency,
+  tokenConversionRate,
 }) {
   const t = useContext(I18nContext)
 
+  const savingAmount =
+    isBestQuote && savings?.total
+      ? formatCurrency(
+          decEthToConvertedCurrency(
+            savings.total,
+            currentCurrency,
+            conversionRate,
+          ),
+          currentCurrency,
+        )
+      : null
+  const savingsIsPositive =
+    savings?.total && new BigNumber(savings.total, 10).gt(0)
+
+  const inDevelopment = process.env.NODE_ENV === 'development'
+  const shouldDisplaySavings =
+    inDevelopment && isBestQuote && tokenConversionRate && savingsIsPositive
+
+  let savingsText
```

Comment:
If we initialize `savingsText` to `''`, we can void the last `else` and possibly the `if (!inDevelopment)` block.

Reply:
Done in 0c0dffdef